data.tex

```diff
@@ -1,27 +1,71 @@
+
+Each pre-operations data preview and survey data release will be accompanied by its own release-specific DPDD, giving e.g. the  database schema for the catalogs included in that dataset.
+For an example data release DPDD, see the online DP0.2 documentation.\footnote{\url{https://dp0-2.lsst.io/data-products-dp0-2/}}
+
 The Rubin data rights policy is described in  \cite{RDO-013}.
 
 \subsection{Prompt data products}
 
-Prompt data products are described in detail in the Data Products Definition Document (\DPDD).
-Alert packets are triggered by difference image source detections and transmitted to community alert brokers\footnote{See \url{https://www.lsst.org/scientists/alert-brokers} for the list of selected brokers.} and are publicly available. 
+Data products for transients, variables, and moving objects will be primarily produced by the Prompt Processing pipelines, which will perform reduction, calibration, difference image analysis (DIA), source detection and measurement, and alert distribution within 60 seconds of image readout. 
+They include images, catalogs and alerts. 
+
+Prompt image data products will be made available 80 hours following camera readout. 
+They include raw images, processed single visit images, difference images, and template images. 
+Prompt image products are proprietary, except for the alert postage stamps distributed in the alert stream and recorded in the alert database.
+Catalog data producs will be available after 24 hours. 
+The image differencing source (\DIASource), forced photometry (\DIAForcedSource), and object (\DIAObject and \SSObject)
+catalog data products are all public. 
+Access to both prompt image and catalog data products is via the Rubin Science Platform (\S~\ref{ssec:dataaccess}) and queryable using VO interfaces. 
+\DIASource, \DIAForcedSource, \DIAObject, and \SSObject catalogs are queryable using VO interfaces to the Prompt Products Database.
+
+Alert are triggered by sources detected with SNR>5 via difference image analysis (DIA) and transmitted to community alert brokers for the list of selected brokers.
+The real-time alert stream is public. 
 Similarly, daily Solar System Processing identifies new Solar System Objects from difference image sources and reports those publicly to the Minor Planet Center.
 
-Catalog and image products, as well as services for running user-generated processing on the data, are available to Rubin Data Rights holders after 24 hours through the Rubin Science Platform (\S~\ref{ssec:dataaccess}).
+Catalog data products, as well as services for running user-generated processing on the data, are available to Rubin Data Rights holders after 24 hours through the Rubin Science Platform (\S~\ref{ssec:dataaccess}).
 \DIASource, \DIAObject, and \SSObject catalogs are queryable using VO interfaces to the Prompt Products Database.
+Image products are available after 80 hours.
```

Comment:
this paragraph repeats much of what is above

Reply:
arrgh - forgot to remove the older text when I rewrote this part. Consolidated now.